### PR TITLE
WMS ID:1700 Updated instructions and screen captures

### PR DESCRIPTION
**Pull request by shppra**
WMS ID:1700 Updated instructions and screen captures

### CODE REVIEW

**Review by shppra — COMMENTED** [review]
added new image

**Review by shppra — COMMENTED** [review]
new instructions with image has been added.

**Review by shppra — COMMENTED** [review]
new instructions with images has been added based on the workshop qa testing.

File: enterprise-manager/emcc/em-configuration-compliance-management/em-configuration-compliance-management.md [hunk 1]
```diff
@@ -345,15 +353,24 @@ In this workshop, you will learn about continuous configuration drift monitoring
     - Not equal sign indicates parameter properties are different across the reference and target compared
     - A red box with 1 (left only) means that the comparison did not find a matching item to compare, this means 2nd target doesn’t have a property configured to compare
     - A red box 2 (right only) means that the comparison did not find a matching item to compare to the second configuration
+
 23. Go to Enterprise --> Configuration --> Comparison & Drift Management Dashboard page, Click on **Drift Results**
 
-  ![comparison-and-drift-management-dashboard-page](images/comparison-and-drift-management-dashboard.png " comparison-and-drift-management-dashboard-page ")
+  ![comparison-and-drift-management-dashboard-page](images/comparison-and-drifts-management-dashboard.png " comparison-and-drift-management-dashboard-page ")
```
Comment (shppra) [inline enterprise-manager/emcc/em-configuration-compliance-management/em-configuration-compliance-management.md:359]: screen capture name has changes to support functionality

Comment (shppra) [reply enterprise-manager/emcc/em-configuration-compliance-management/em-configuration-compliance-management.md:359]: Added new instructions

**Review by shppra — COMMENTED** [review]
Added changes to instructions and screen captures

**Comment by shppra** [comment]
Could you unblock my merge.

**Review by shppra — COMMENTED** [review]
Reviewed all my changes

**Review by shppra — COMMENTED** [review]
Can you approve it.

**Comment by shppra** [comment]
Please review at earliest convenience

**Review by arabellayao — APPROVED** [review]

**Pull request merged**